error.go

```diff
@@ -121,6 +121,16 @@ func (une *UnknownNetworkError) Error() string {
 // NotFound denotes the type of this error
 func (une *UnknownNetworkError) NotFound() {}
 
+// EndpointNameError is returned when a endpoint with the same name already exists in that network.
+type EndpointNameError string
+
+func (epe EndpointNameError) Error() string {
+	return fmt.Sprintf("service endpoint with name %s already exists", string(epe))
+}
+
+// Forbidden denotes the type of this error
+func (epe EndpointNameError) Forbidden() {}
+
```

Comment:
yeah, that is the idea. It's that I did not have time to remove the old errors, sorry.